Content.Server/GameObjects/Components/Destructible/Threshold.cs

```diff
@@ -16,6 +20,26 @@ namespace Content.Server.GameObjects.Components.Destructible
 {
     public class Threshold : IExposeData
     {
+        [ViewVariables] public int? Damage;
```

Comment:
With this architecture, it's not clear to me how this would work if an entity had multiple thresholds defined for different damage classes. With the old approach it was very clear.

It's now possible for an entity to simultaneously satisfy multiple thresholds. For example, if I put a 20 Brute threshold and 20 Burn threshold on an entity, what happens when they have both? If each has an appearance change to the base sprite then what is the new appearance supposed to be? I could see this being confusing to people trying to define thresholds on their objects, unless there's a very easy way to understand how it works when multiple damage types / classes are involved.

Reply:
Fixed in https://github.com/space-wizards/space-station-14/pull/3102